lib/mongodb.js

```diff
@@ -797,6 +797,33 @@ function hasNearFilter(where) {
   return false;
 }
 
+MongoDB.prototype.buildLookup = function (relation, include, model) {
+  var self = this;
```

Comment:
I think that will work we will just include the relations we wish to aggregate inside options
http://loopback.io/doc/en/lb2/Model-definition-JSON-file.html#data-source-specific-options
may be we can include mongodb aggregated relations only inside mongodb options.